KancolleVerificationTeam/KCVDB.Client
# Separated Diff to another assembly so the main library won't depend on protobuf-net.

[Fixes]
 - Separate Diff sender to another assembly
 - Clean-up code in KCVDB.Client
 - Update KanColleDbPost.Form1 to obtain an API client with DfiffDataApiSender

[Tests]
 - Start KanColleDbPost, launch KanColle, verify the app shows diff logs properly.
 - Run TestClient, verify it obtains QueueingClient with RawApiDataSender.
 - Run TestClient, verify it successfully sends data to KCVDB API server.
